src/domains.rs

```diff
@@ -4,13 +4,27 @@ use ssz_rs::prelude::*;
 pub enum DomainType {
     BeaconProposer,
     BeaconAttester,
-    RANDAO,
+    Randao,
     Deposit,
     VoluntaryExit,
     SelectionProof,
     AggregateAndProof,
 }
 
+impl DomainType {
+    pub fn get_domain_constant(&self) -> [u8; 4] {
+        match self {
+            DomainType::BeaconProposer => 0u32.to_le_bytes(),
+            DomainType::BeaconAttester => 1u32.to_le_bytes(),
+            DomainType::Randao => 2u32.to_le_bytes(),
+            DomainType::Deposit => 3u32.to_le_bytes(),
+            DomainType::VoluntaryExit => 4u32.to_le_bytes(),
+            DomainType::SelectionProof => 5u32.to_le_bytes(),
+            DomainType::AggregateAndProof => 6u32.to_le_bytes(),
+        }
+    }
+}
+
```

Comment:
what do you think about something like this?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=744eaf45903e77bce6c7b41d2889d8e8

Reply:
fixed